parser/parser.py

```diff
@@ -348,12 +350,11 @@ def process_class_file(country, class_file_src_path, class_file_target_path, cla
 
         # Prettify the code
         prettified_code = prettify_xml(str(dita_soup))
-
         with open(file_path, "wb") as f:
             f.write(prettified_code.encode("utf-8"))
 
 
-def process_category_pages(country, category_page_link, country_name, country_flag_link, root_path):
+def process_category_pages(country, category_page_link, category, country_name, country_flag_link, root_path):
```

Comment:
hmm, is the `category` (that came from the link title), or the `folder name` (that came from the URL)?   I think we should support both, since the first on is used for titles on the screen, and the second one is used for parsing the legacy content.
I _think_ `category_folder` is the correct name for the second one.

Reply:
The category is coming from the links title, i am taking it from this line 

https://github.com/DeepBlueCLtd/LegacyMan/blob/53bdf388f72d31b595ef2bc18f3a294f46d0139e/data/Britain/Britain1.html#L36